LSP.sublime-settings

```diff
@@ -305,12 +316,6 @@
       ],
       "selector": "source.reason | source.ocaml",
     },
-    "gopls": {
-      "command": [
-        "gopls"
-      ],
-      "selector": "source.go",
-    },
```

Comment:
I used to announce this removal in a previous release message. Make sure to announce this in the current release message. Now that I’m writing this, I’m not sure if it’s a good idea to remove this without an announcement message.

Reply:
Because it may break current user’s workflow.

Reply:
Added changelog listing the breaking changes.

It will be updated before release to list all other changes.